spec/spec_helper.rb

```diff
@@ -25,7 +25,6 @@
   SimpleCov::Formatter::HTMLFormatter,
   SimpleCov::Formatter::LcovFormatter
 ])
-SimpleCov.start
```

Comment:
Is this not needed anymore / is it being started somewhere else?

Reply:
This is included in https://github.com/hathitrust/holdings-backend/blob/main/.simplecov